pajbot/migration_revisions/db/0004_unify_user_model.py

```diff
@@ -118,16 +226,19 @@ def safe_to_int(input):
         # create Server-side cursor
```

Comment:
no need for `if users_count > 0` anymore, it was meant so we don't show a progress bar on new databases